docs/operator-manual/rbac.md

```diff
@@ -23,26 +23,26 @@ These default built-in role definitions can be seen in [builtin-policy.csv](http
 
 When a user is authenticated in Argo CD, it will be granted the role specified in `policy.default`.
 
-!!! warning "Denying Default Permissions"
+!!! warning "Restricting Default Permissions"
 
-    **All authenticated users get _at least_ the permissions granted by the default policy. This access cannot be blocked
+    **All authenticated users get _at least_ the permissions granted by the default policies. This access cannot be blocked
     by a `deny` rule.** It is recommended to create a new `role:authenticated` with the minimum set of permissions possible,
     then grant permissions to individual roles as needed.
 
 ## Anonymous Access
 
-Enabling anonymous access to the Argo CD instance allows users to assume the default role permissions specified by `policy.default` **without being authenticated**.
+Enabling anonymous access to the Argo CD instance allows users to assume the default role policies specified by `policy.default` **without being authenticated**.
```

Comment:
It seems that here you are referring to the general term? In this case, I think that it should remain as "permission"